chronodialogpreference/src/main/res/layout/chrono_preference.xml

```diff
@@ -0,0 +1,10 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="19dp" />
```

Comment:
`19dp` doesn't support many Android devices.
Can I change from `19dp` to `16dp` or `20dp` ?

Reply:
@koji-1009 Not sure what do you mean. `dp` are device-independent units. The value of `19dp` was hand-adjusted to look the best. That said, `20dp` won't make a big difference...

Reply:
@ZimM-LostPolygon 
Most android devices have a screen width and height that is a multiple of 4.
For this reason, it is better to use multiples of 4 and 8 so that the view is displayed correctly when the user enlarges the screen.

Check "Spacing methods" at Material Design Guideline.
https://material.io/design/layout/spacing-methods.html